### PR TITLE
Docs/restructure

### PR DESCRIPTION
**Pull request by aleksei-korolev**
# Description of change

Restructure the documentation to the new guidelines.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## How the change has been tested

Local wiki setup, and I clicked through links.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aleksei-korolev** [comment]
@lucas-tortora @Dr-Electron

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
Guess you can remove `./tutorials/README.md`, `./internal/README.md` and update `./README.md`. Also let's move `charts.pptx` one folder up. Maybe create a folder like `./resources` and but it in there?

**Review by Dr-Electron — DISMISSED** [review]
Lgtm

File: documentation/docs/getting_started/python.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ keywords:
...
 - pip
 - unpack
-- reference
+- getting started
 ---
-# Getting Started with IOTA Wallet Python Binding
+# Getting Started with Python
 
-## Security
+## Securityhttps://wiki.iota.org/chrysalis-docs/guides/backup_security
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/python.md]: I think this link may be misplaced or malformatted.

**Review by lucas-tortora — CHANGES_REQUESTED** [review]
Just 1 small comment, other than that this LGTM! :)

**Review by lucas-tortora — DISMISSED** [review]

File: documentation/docusaurus.config.js [hunk 2]
```diff
@@ -21,4 +21,5 @@ module.exports = {
     ],
   ],
   staticDirectories: [path.resolve(__dirname, 'static')],
-};
+  onBrokenLinks: 'warn',
```
Comment (Dr-Electron) [inline documentation/docusaurus.config.js]: You can remove this. The wiki or cli will do it

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
Looks good to me.

One last thing.
Can you move the `chart.pptx` file into `documentation/resources/charts.pptx`?

File: documentation/docs/reference/python.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ keywords:
...
 - reference
 ---
-# API Reference
+# Python API Reference
 
 :::info
 The following APIs will throw an exception if an error occurs.
-For all the optional values, the default values are the same as the ones in the [Rust API](../rust/api_reference.md).
+For all optional values, the default values are the same as in the [Rust API](rust.md).
 :::
```
Comment (Thoralf-M) [inline documentation/docs/reference/python.md:17]: Not related to this change, but there is 
```
node = {
    'url': string,
    'auth': [NodeAuth](#nodeauth) (optional),
}
```
which should be 
```
node = {
    'url': string,
    'auth': [NodeAuth](#nodeauth) (optional),
    'disabled': bool,
}
```
could you update this in this PR?

Comment (Thoralf-M) [reply documentation/docs/reference/python.md:17]: Thanks :heart:

**Review by Thoralf-M — COMMENTED** [review]
For what does `documentation/docs/tutorials/.keep` exists?

**Comment by aleksei-korolev** [comment]
> For what does `documentation/docs/tutorials/.keep` exists?

Too keep the folder in git to make it clearer where to put tutorials once we have some.

**Review by Thoralf-M — APPROVED** [review]

File: documentation/sidebars.js [hunk 4]
```diff
@@ -10,128 +10,125 @@
...
-    id: 'troubleshooting',
-    label: 'Troubleshooting'
-  },
-  {
-    type: 'doc',
-    id: 'contribute',
-    label: 'Contribute',
-  }
   ]
 };
```
Comment (Dr-Electron) [inline documentation/sidebars.js:134]:
```suggestion
};

```

New line. Please. I can't see this GitHub symbol for missing new lines anymore :trollface:

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**